.github/workflows/make-release.yml

```diff
@@ -0,0 +1,97 @@
+name: make-release
+
+on:
+  workflow_dispatch:
+    inputs:
+      target_branch:
+        description: >-
+          JSON array of branches to target. The first branch in the array will
+          be marked as the latest release.
+        required: false
+        type: string
+        default: '["release/0.6","release/0.5"]'
+
+jobs:
+  bump-version:
+    strategy:
+      matrix:
+        target_branch: ${{ fromJSON(inputs.target_branch) }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: '${{ matrix.target_branch }}'
+          token: '${{ secrets.DIVVIUP_GITHUB_AUTOMATION_RELEASE_PAT }}'
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+
+      - name: Restore cargo-edit from cache
+        id: cache-cargo-edit-restore
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+            ~/.cargo/bin
+          key: ${{ runner.os }}-cargo-edit
+
+      - run: cargo install cargo-edit --no-default-features --features set-version
+        if: steps.cache-cargo-edit-restore.outputs.cache-hit != 'true'
```

Comment:
Barring cache eviction, what will allow us to upgrade to newer versions of `cargo-edit`? (If it's nothing, WDYT about just reinstalling `cargo-edit` each time -- at least on my machine, it only takes a few seconds to compile & install. This isn't such a good suggestion if the install takes much longer on GH's actions runners.)

Reply:
We could write the version of cargo-edit into the workflow, pass it to cargo install, and put it in the cache key, as we do with a couple other Rust utilities. I expect the main reason we'll want to upgrade cargo-edit is to keep up with Cargo manifest format additions.

Reply:
It does take about 3 minutes on a runner, which is an annoying amount of time when iterating on the action, but I suppose not a big deal given the whole release CI will take about an hour.

Either way I was figuring we can just do `gh cache delete Linux-cargo-edit` when we want to flush the cache and get a new version.

Reply:
That's a good idea, I see we do that in https://github.com/divviup/libprio-rs/blob/main/.github/workflows/supply-chain.yml. I'll adopt this.

Reply:
Hm, yeah, I think I like the idea of fixing the version of `cargo-edit` (following the more general principle that the versions of dependencies should be fixed).